packages/app/tests/Charcoal/App/ServiceProvider/FilesystemServiceProviderTest.php

```diff
@@ -207,7 +207,7 @@ public function testConfigWithoutTypeThrowsException()
 
     private function createAppConfig($defaults = null)
     {
-        return new AppConfig($defaults);
+        return new AppConfig(array_replace(['base_path' => sys_get_temp_dir()], $defaults));
```

Comment:
Improve readability.

```suggestion
        return new AppConfig(array_replace([
            'base_path' => sys_get_temp_dir(),
        ], $defaults));
```

Reply:
Done